applications/serial_lte_modem/doc/FOTA_AT_commands.rst

```diff
@@ -17,110 +17,94 @@ The ``#XFOTA`` command sends various types of FOTA requests, based on specific o
 Set command
 -----------
 
-The set command allows you to send a FOTA request.
+The set command performs FOTA-related operations.
 
 Syntax
 ~~~~~~
 
 ::
 
-   AT#XFOTA=<op>,<file_url>[,<sec_tag>[,<pdn_id>]]
+   AT#XFOTA=<op>[,<file_url>[,<sec_tag>[,<pdn_id>]]]
 
-* The ``<op>`` parameter can accept one of the following values:
+* The ``<op>`` parameter must be one of the following values:
 
   * ``0`` - Cancel FOTA (during download only).
   * ``1`` - Start FOTA for application update.
   * ``2`` - Start FOTA for modem delta update.
-  * ``6`` - Read application image size and version (optional for application FOTA).
-  * ``7`` - Read modem scratch space size and offset (optional for modem FOTA).
-  * ``8`` - Erase MCUboot secondary slot (optional for application FOTA).
-  * ``9`` - Erase modem scratch space (optional for modem FOTA).
+  * ``7`` - Read modem DFU area size and firmware image offset.
+  * ``9`` - Erase modem DFU area.
 
 * The ``<file url>`` parameter is a string.
   It represents the full HTTP or HTTPS path of the target image to download.
+  It must be provided to the start operations.
 * The ``<sec_tag>`` parameter is an integer.
   It indicates to the modem the credential of the security tag used for establishing a secure connection for downloading the image.
   It is associated with the certificate or PSK.
-  Specifying the ``<sec_tag>`` is mandatory when using HTTPS.
+  It must be provided to the start operations when using HTTPS.
 * The ``<pdn_id>`` parameter is an integer.
   It represents the Packet Data Network (PDN) ID that can be used instead of the default PDN for downloading.
+  It can be provided to the start operations.
 
-Response syntax
-~~~~~~~~~~~~~~~
-
-::
+.. note::
+      Erasing the the modem DFU area (when doing modem FOTA) is optional in that
+      the update process will automatically take care of doing it if it is needed.
 
-  #XFOTA: <primary_partition_id>,<MCUBOOT_version>,<img_size>,<img_version>
-  #XFOTA: <secondary_partition_id>,<MCUBOOT_version>,<img_size>,<img_version>
+      However, the downside to this is that the command starting the update will take longer to complete.
+      Also, it will leave the connection to the FOTA server idle while performing the erasure, which could provoke issues.
```

Comment:
```suggestion
   When doing modem FOTA, erasing the modem DFU area is optional since the update process will automatically erase the area if needed.
   However, this leads to the command starting the update taking longer to complete, and also leaves the connection to the FOTA server idle while the area is being erased, which can provoke issues.